chain/messagepool.go

```diff
@@ -73,6 +85,9 @@ func (ms *msgSet) add(m *types.SignedMessage) error {
 func NewMessagePool(sm *stmgr.StateManager, ps *pubsub.PubSub) *MessagePool {
 	cache, _ := lru.New2Q(build.BlsSignatureCacheSize)
 	mp := &MessagePool{
+		closer:        make(chan struct{}),
+		repubTk:       time.NewTicker(2 * time.Minute),
```

Comment:
Maybe `10 * build.BlockDelay * time.Second`?

Reply:
10*30s will be 5 min, sgwm.